### ISSUE TITLE
WCAG 2.5.3,New for WCAG 2.1: Ensures that elements labelled through their content must have their visible text as part of their accessible name (#institution-menu)

### ISSUE THREAD

**Issue by zcjess** [zcjess/Ecoach-accessibility-issues#4]
#### Issue
`Ensures that elements labelled through their content must have their visible text as part of their accessible name` ([label-content-name-mismatch](https://dequeuniversity.com/rules/axe/4.6/label-content-name-mismatch?application=msftAI))
#### Target application
[ECoach | STATS250 Home](https://umecoach.miserver.it.umich.edu/coach/1/home/)
#### Element path
#institution-menu
#### Snippet
`<button class="dropdown-toggle btn btn-navy" id="institution-menu" role="button" data-target="#" data-toggle="dropdown" type="button" aria-label="Institutions toggle">
        Dashboard
    </button>`
#### How to fix
    Fix any of the following:
      Text inside the element is not included in the accessible name
#### Environment
Chrome version 110.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.37.3 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
<img width="164" alt="image" src="https://user-images.githubusercontent.com/90011962/223473576-9f9cc170-1b18-4b78-ab14-03a9f06f85bf.png">

**Comment by zcjess** [comment]
The element's role is missing or is not appropriate for the element's function.

Role: link -> button
<img width="212" alt="image" src="https://user-images.githubusercontent.com/90011962/226637723-22146ce5-739f-41e1-87ec-e2dc2a0be9a0.png">

Role: link -> tab
<img width="314" alt="image" src="https://user-images.githubusercontent.com/90011962/226643387-5ca8bd81-3484-4084-ba59-5ff899a692c6.png">

Role: link -> grid?
<img width="672" alt="image" src="https://user-images.githubusercontent.com/90011962/226647573-3a051033-c4c3-4562-9fe3-72cdcbc1f89c.png">

Role: link -> button, many same-looking buttons
<img width="141" alt="image" src="https://user-images.githubusercontent.com/90011962/226649177-cdd75ad2-ae42-4700-9483-aaefdd10b1ff.png">
